addons/website/static/src/scss/website.scss

```diff
@@ -1460,6 +1460,10 @@ header {
             vertical-align: middle;
         }
     }
+
+    .navbar-collapse.show & .dropdown-menu {
+        position: static;
+    }
```

Comment:
This is what I get while testing:

https://github.com/odoo/odoo/assets/25032876/7873624b-e0dc-4bbf-b52b-127fd3113558

Maybe you just missed a `display: block` on the `.dropdown-toggle` :thinking: ?